cmd/README.md

```diff
@@ -8,13 +8,13 @@ Command Line tool for managing the WSO2 Micro Integrator
 
 - ### Setting up the development environment
     1. Install [Go 1.12.x](https://golang.org/dl)
-    2. Fork the [repository](https://github.com/wso2/micro-integrator)
+    2. Fork the [repository](https://github.com/wso2/product-mi-tooling)
     3. Clone your fork into any directory
-    5. `cd` into cloned directory and then cd into `micro-integrator/cmd`
+    5. Navigate into cloned directory and then navigate into `product-mi-tooling/cmd`
     6. Execute `go mod vendor` or `go mod download` to download all the dependencies
     
 - ### Building from source
-    cd into micro-integrator home directory
+    Navigate into product-mi-tooling home directory
 
     Execute `make install` to build both the **Micro Integrator** and **MIcro Integrator CLI** at once.
```

Comment:
This also has to be updated as we have only CLI now .

Reply:
Fixed